src/nirfsg/metadata/functions.py

```diff
@@ -4679,196 +4693,14 @@
                 'use_in_python_api': True
             }
         ],
-        'returns': 'ViStatus',
-        'use_session_lock': False
-    },
-    'SetWaveformBurstStartLocations': {
```

Comment:
This method is not deleted in the latest internal metadata export. Nor do you seem to have any PRs where this is set to not codegen. Did you hand-edit this file?